spec.md

```diff
@@ -565,15 +565,18 @@ message VolumeMetadata {
 }
 
 // A standard way to encode credential data. The total bytes of the values in
-// the Data field must be less than MaxSecretSize bytes.
+// the Data field must be less than 1 Mebibyte.
```

Comment:
i don't think we need to address that in the PR. maybe something to follow up with.